solidity/contracts/deposit/DepositUtils.sol

```diff
@@ -438,9 +438,12 @@ library DepositUtils {
     /// @return             True if successful, otherwise revert.
     function pushFundsToKeepGroup(Deposit storage _d, uint256 _ethValue) internal returns (bool) {
         require(address(this).balance >= _ethValue, "Not enough funds to send");
-        IBondedECDSAKeep _keep = IBondedECDSAKeep(_d.keepAddress);
-        _keep.returnPartialSignerBonds.value(_ethValue)();
-        return true;
+        if(_ethValue > 0){
+            IBondedECDSAKeep _keep = IBondedECDSAKeep(_d.keepAddress);
+            _keep.returnPartialSignerBonds.value(_ethValue)();
+            return true;
+        }
+        return false;
```

Comment:
This is going to fall under the return thing we're doing in the other PR, but slightly different: this function only used to ever return true… But also no one pays attention to the return value. I think we can drop the return value.